signInAndSignOut_feature/UI/PackagesViewController/PackagesViewController.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+
+class PackagesViewController: BaseViewController {
+    
+    // MARK: - IBOutlet -
+    @IBOutlet weak var tableViewPackages: UITableView!
+    
+    // MARK: - Variables -
+    var packagesArr = [Packages]()
+    
+    // MARK: - Life Cycle -
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updateDataArr()
+    }
+    // MARK: - OVERRIDE FUNCTION -
+    override func viewWillAppear(_ animated: Bool){
+        self.navigationController?.navigationBar.isHidden = true
+    }
+    override func viewWillDisappear(_ animated: Bool) {
+        self.navigationController?.navigationBar.isHidden = false
+    }
+    // MARK: - IBAction -
+    
+    @IBAction func directTodashboardbtn(_ sender: Any) {
+        directTodashboard()
+    }
+    
+    // MARK: - Methods -
+    private func updateDataArr() {
+        tableViewPackages.register(UINib(nibName: "PackageCell", bundle: nil), forCellReuseIdentifier: "packagecell")
```

Comment:
Still you're passing the same image on every package

Reply:
updated